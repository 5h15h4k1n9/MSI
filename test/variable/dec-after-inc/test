increment()
increment()
decrement()
count
exit
