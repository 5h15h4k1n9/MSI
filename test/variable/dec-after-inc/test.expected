(void) void
(void) void
(void) void
(int) 1
Bye-bye!
